### PR TITLE
Create contact us section

### LINKED ISSUE #14: Create contact us section
- [ ] Title
- [ ] Form
- [ ] Submit button

### PR DESCRIPTION
**Pull request by hussambanna**
related to #14

### CODE REVIEW

**Review by ahmedirheem — APPROVED** [review]

**Review by abeerAyad — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -128,6 +128,20 @@ <h4>Suzaan clarke</h4>
...
+<!-- Start Contact us Section -->
+<section id="contact-us">
+  <div class="container">
+      <div class="title"><h2>Contact Us</h2></div>
+      <form>
+          <input type="text" class="field" placeholder="Your Name">
+          <input type="text" class="field" placeholder="Your Email">
+          <textarea placeholder="Your Message" class="field"></textarea>
+          <button class="btn">Submit</button>
+      </form>
```
Comment (yaqootturman) [inline index.html]: add some functionality for these elements
suggestions: 
- after submitting a message, create a pop-up message to thank the user for his feedback
- if the user leaves a message field empty give him a message or red flag to confirm the form contain a message

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -128,6 +128,26 @@ <h4>Suzaan clarke</h4>
...
+      <form>
+          <input type="text" id="name" class="field" placeholder="Your Name">
+          <input type="text" id="email" class="field" placeholder="Your Email">
+          <textarea id="message" placeholder="Your Message" class="field"></textarea>
+          <button class="submitBtn">Submit</button>
+      </form>
+  </div>
+</section>
+
+<div class="messageThank">
```
Comment (yaqootturman) [inline index.html]: make the name as your convention for CSS which is snake-case

File: index.html [hunk 3]
```diff
@@ -128,6 +128,26 @@ <h4>Suzaan clarke</h4>
...
 
+<!-- Start Contact us Section -->
+<section id="contact-us">
+  <div class="container">
+      <div class="title"><h2>Contact Us</h2></div>
+      <form>
+          <input type="text" id="name" class="field" placeholder="Your Name">
+          <input type="text" id="email" class="field" placeholder="Your Email">
+          <textarea id="message" placeholder="Your Message" class="field"></textarea>
+          <button class="submitBtn">Submit</button>
```
Comment (yaqootturman) [inline index.html:139]: make the name as your convention for CSS which is snake-case

File: index.html (hunk 2 above)
Comment (yaqootturman) [reply index.html]: Add a style for the pop up message to make it smaller

**Review by yaqootturman — CHANGES_REQUESTED** [review]
I can't see where you used a ContactUs.png , If you dont use it, delete it, please!

File: index.html (hunk 2 above)
Comment (hussambanna) [reply index.html]: Do you mean like that 'message_thank' ?

File: home/index.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ const newSlides = () => {
 newSlides();
 
 let textarea = document.querySelector("textarea");
-let messageThank = document.querySelector(".messageThank");
+let messageThank = document.querySelector(".message_thank");
```
Comment (yaqootturman) [inline home/index.js]: this is not snake, the snake is message-thank 
all your new CSS names of classes need to change btw

File: index.html [hunk 5]
```diff
@@ -133,15 +133,15 @@ <h4>Suzaan clarke</h4>
   <div class="container">
       <div class="title"><h2>Contact Us</h2></div>
       <form>
-          <input type="text" id="name" class="field" placeholder="Your Name">
-          <input type="text" id="email" class="field" placeholder="Your Email">
-          <textarea id="message" placeholder="Your Message" class="field"></textarea>
+          <input type="text" class="field" placeholder="Your Name">
```
Comment (yaqootturman) [inline index.html]: make the class name more descriptive, just as before: name and email, and if you want add same style for them grouping them in css

File: index.html [hunk 6]
```diff
@@ -133,15 +133,15 @@ <h4>Suzaan clarke</h4>
...
+          <input type="text"  class="field" placeholder="Your Email">
+          <textarea  placeholder="Your Message" class="field"></textarea>
           <button class="submitBtn">Submit</button>
```
Comment (yaqootturman) [inline index.html:139]: and same here, this what i want you to change in my previous review,

File: home/index.css [hunk 7]
```diff
@@ -225,7 +225,7 @@ nav {
 #contact-us {
   height: 480px;
   padding: 10% 0;
-  background: url(/assets/ContactUs.png);
+  background: url("/assets/ContactUs.png");
```
Comment (yaqootturman) [inline home/index.css]: the image should called contactUs, as other files naming in your code , you cant ignore the style of namig, so change it here and change the file name itself

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (hussambanna) [reply index.html]: Done

File: index.html (hunk 6 above)
Comment (hussambanna) [reply index.html:139]: Done

File: home/index.css (hunk 7 above)
Comment (hussambanna) [reply home/index.css]: Done

File: home/index.js (hunk 4 above)
Comment (hussambanna) [reply home/index.js]: Done

**Review by yaqootturman — APPROVED** [review]

**Review by abeerAyad — APPROVED** [review]

**Pull request merged**